src/components/Link/Link.tsx

```diff
@@ -1,12 +1,21 @@
-import React, { forwardRef } from "react";
-import PropTypes from "prop-types";
+import React, { forwardRef, FC, ReactElement } from "react";
 import cn from "classnames";
 import { Link as OriginalLink } from "react-router-dom";
 
-const Link = forwardRef(
-  ({ to, children, secondary, tertiary, className, ...props }, ref) => {
+interface Props {
+  to: string,
+  children: HTMLElement,
+  className: string,
+  secondary: boolean,
+  tertiary: boolean,
+  rel?: string,
+  target?: string
+}
+
+const Link: FC<Props> = forwardRef<HTMLAnchorElement, Props>(
+  ({ to, children, secondary, tertiary, className, ...props }: Props, ref): ReactElement => {
     let isExternal = false;
-    let classes = className;
+    let classes: string = className;
```

Comment:
Actually the `className` can be anything there. The `classnames` module accept an `object`, `array` or `string` as arguments.

Reply:
yes, but the output of the classnames will be always a string

Reply:
That's true. I was more thinking of this use case:

```
<Link to="/apps" className={{ myClass: displayOnlyIfThisVariableIsTrue }} />
```

But I guess forcing the developer to use `classnames` while calling the component also makes sense.